karma.conf.js

```diff
@@ -0,0 +1,49 @@
+var webpackConfig = require("./webpack.config");
+Object.assign(webpackConfig, {
+    debug: true,
+    devtool: "inline-source-map",
+    externals: webpackConfig.externals.concat([
+        "react/lib/ExecutionEnvironment",
+        "react/lib/ReactContext",
+        "react/addons",
+        "jsdom"
+    ]),
+    postLoaders: [ {
+        test: /\.ts$/,
+        loader: "istanbul-instrumenter",
+        include: path.resolve(__dirname, "src"),
+        exclude: /\.(spec)\.ts$/
+    } ]
+});
+
+module.exports = function(config) {
+    config.set({
+        basePath: "",
+        frameworks: [ "jasmine" ],
+
+        files: [
+            { pattern: "src/**/*.ts", watched: true, included: false, served: false },
```

Comment:
webpack is handling the inclusions of the files so all the configs three parameters are not need. see commit